src/ProjectOrigin.WalletSystem.IntegrationTests/ApiTests/ApiTests.cs

```diff
@@ -113,7 +138,9 @@ public async Task can_query_certificates_cursor()
 
         //Act
         var res = await httpClient.GetAsync($"v1/certificates/cursor?UpdatedSince={updatedSince}");
-        var content = JsonConvert.DeserializeObject<ResultList<GranularCertificate, PageInfoCursor>>(await res.Content.ReadAsStringAsync());
+        var content =
+            JsonConvert.DeserializeObject<ResultList<GranularCertificate, PageInfoCursor>>(
+                await res.Content.ReadAsStringAsync());
```

Comment:
why are you correcting these? it's easier to read when they are one line :)

Reply:
my formatter. Fixed